scripts/create_gct_file.R

```diff
@@ -67,24 +67,27 @@ df <- data.frame("Name" = df[ ,1], "Description" = rep(NA, nrow(df)),
 num.samples <- ncol(df) - 2
 num.genes <- nrow(df)
 
-# Make .gct specific Header with the number of genes and samples in the second 
-# line. 
+# Make .gct specific Header with the number of genes and samples in the second
+# line.
 header <- c("#1.2", paste0(c(num.genes, num.samples), collapse = "\t"))
 
-  # Produce error if output file already exists. 
+  # Produce error if output file already exists.
 if (file.exists(opt$output)) {
   if(opt$rewrite){
     warning(paste0("Overwriting file named '", opt$output, "'"))
   } else {
-    stop(paste(opt$output, "already exists. 
+    stop(paste(opt$output, "already exists.
         Use '--rewrite' option if you want this file to be overwritten."))
   }
 }
 # Write the header and then append the data to the header
 write(header, file = opt$output)
 
-# write.table will throw a warning that we are appending column names to the file, 
-# but that's what we want to do since we need that header at the top for GenePattern 
+# write.table will throw a warning that we are appending column names to the file,
+# but that's what we want to do since we need that header at the top for GenePattern
 # to recognize it so we suppress that warning
 suppressWarnings(write.table(df, row.names = FALSE, opt$output, append = TRUE,
                              quote = FALSE, sep="\t"))
+
+# Print out message so the user knows the file is made
+message(paste("Congrats. Your .gct file has been made and is called:", opt$output))
```

Comment:
> Your .gct file <file_name> has been created! You can find it at <output path>. 

I would remove the congrats, since it's a mid step and not end of their analysis.